### DISCUSSION TITLE
Dynamic Update Server during installation new update getting error

### DISCUSSION THREAD

**Discussion by ankitverma304** [tauri-apps/tauri#9703] [Q&A]
Hi Team,
I am trying to create new windows app using Tauri with react js.
I am using Dynamic Update Server feature for new update.
1. Implementation are done
2. Update popup is occurred
3. After click on yes getting error: Signature decoding error: Invalid byte 58, offset 5
Please help me to sort out the issue, or suggest where I am doing wrong.

**Comment by FabianLars** [comment]
if i remember correctly, this error most often is that the "signature" field in the server response is not the full **content** of the .sig file of the updater bundle.

**Reply by ankitverma304** [reply]
As I notice, this error also occur before creating .sig file, while using command: npm run tauri build
Command final response:
Finished 2 bundles at:
        D:\windows_app\My\testing\first-app\httpdocs\src-tauri\target\release\bundle\msi\first app_1.0.1_x64_en-US.msi
        D:\windows_app\My\testing\first-app\httpdocs\src-tauri\target\release\bundle\nsis\first app_1.0.1_x64-setup.exe
        D:\windows_app\My\testing\first-app\httpdocs\src-tauri\target\release\bundle\msi\first app_1.0.1_x64_en-US.msi.zip (updater)
        D:\windows_app\My\testing\first-app\httpdocs\src-tauri\target\release\bundle\nsis\first app_1.0.1_x64-setup.nsis.zip (updater)

       Error Invalid byte 95, offset 5.

**Reply by misxki** [reply]
I have the same issue, the .sig files are not created when I do `"createUpdaterArtifacts": true` and the build errors with `Error Invalid symbol 61, offset 352.`
If I set it to false, it builds without an error.

Did you find a fix for this?
